modules/registrars/openprovider/Controllers/System/DomainSuggestionsController.php

```diff
@@ -60,7 +60,7 @@ public function get($params)
 
         $args['sensitive'] = isset($suggestionSettings['sensitive']) && $suggestionSettings['sensitive'] == 'on';
 
-        if (isset($suggestionSettings['suggestTlds']) && count($suggestionSettings['suggestTlds']) > 0) {
+        if (isset($suggestionSettings['suggestTlds']) && !empty($suggestionSettings['suggestTlds'])) {
```

Comment:
Seems isset also not needed if we're using empty,
also, make sure always variable type (string or array)